scripts/download/download_mm.py

```diff
@@ -63,3 +64,7 @@
 for chrom in list(range(1, 20)) + ['X', 'Y']:
     filename = 'chr{}.fa.gz'.format(chrom)
```

Comment:
It's just concatenating two lists

```sh
>>> list(range(1,3)) + ['X', 'Y']
[1, 2, 'X', 'Y']
```

I'm creating the filenames to download files from [here](http://hgdownload.cse.ucsc.edu/goldenPath/mm9/chromosomes/).